hack/run-lens.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+which deck &>/dev/null || k8s.io/test-infra/prow/cmd/deck
+
+export KUBECONFIG="$(mktemp)"
+
+cleanup(){
+  rm -f $KUBECONFIG
+  rm -f $deck_config
```

Comment:
Plz run this through shellcheck and quote things :)

Reply:
If anything of these ever contains something that would need quoting the whole thing will break anyways

Reply:
I will not block the PR on this, but in general quoting is bash good form, and it allows using tools like shellcheck to spot more important errors without drowning in warnings.